incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -108,8 +108,8 @@ var _ = Describe("Conversion from v1alpha1 to v1alpha2", func() {
 		MustRun("kubectl annotate ns", nsA, "hnc.x-k8s.io/managed-by=bar")
 		MustRun("kubectl annotate ns", nsA, "hnc.x-k8s.io/managedBy=foo") // deprecated in v0.6
 
-		// Verify that the deprecated version is removed
-		FieldShouldNotContain("ns", "", nsA, ".metadata.annotations", "hnc.x-k8s.io/managedBy:foo")
+		// Verify that the deprecated version is not copied to the current version
+		time.Sleep(1 * time.Second)
```

Comment:
Why we need this 1 second now?

Reply:
Hmm, I thought the next function didn't do retries but clearly it did. Removed.